sugarloaf/src/font/mod.rs

```diff
@@ -275,6 +275,21 @@ impl FontLibraryData {
         let mut db = loader::Database::new();
         db.load_system_fonts();
 
+        let additional_dirs = spec
+            .additional_dirs
+            .unwrap_or_default()
+            .into_iter()
+            .map(PathBuf::from);
+
+        let config_home_dir = std::env::var("RIO_CONFIG_HOME")
```

Comment:
I would avoid add this functionality now, I need to think more about this solution tbh. And prob will be good to release Rio with `RIO_CONFIG_HOME` first.

There's few concerns I have so far regarding env vars usage but most importantly is that sugarloaf shouldn't be connected to rio whatsoever since is used by other projects.